.github/workflows/CI-workflow.yml

```diff
@@ -41,43 +55,86 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
 
       - name: Build and Run Tests
+        id: step-build-test-linux
         run: |
-          export OPENAI_KEY=$(aws secretsmanager get-secret-value --secret-id github_openai_key --query SecretString --output text)
-          export COHERE_KEY=$(aws secretsmanager get-secret-value --secret-id github_cohere_key --query SecretString --output text)
-          echo "::add-mask::$OPENAI_KEY"
-          echo "::add-mask::$COHERE_KEY"
-          ./gradlew build
-      - name: Publish to Maven Local
-        run: |
-          ./gradlew publishToMavenLocal
-      - name: Multi Nodes Integration Testing
-        run: |
-          export OPENAI_KEY=$(aws secretsmanager get-secret-value --secret-id github_openai_key --query SecretString --output text)
-          export COHERE_KEY=$(aws secretsmanager get-secret-value --secret-id github_cohere_key --query SecretString --output text)
-          echo "::add-mask::$OPENAI_KEY"
-          echo "::add-mask::$COHERE_KEY"
-          ./gradlew integTest -PnumNodes=3
+          chown -R 1000:1000 `pwd`
+          su `id -un 1000` -c 'whoami && java -version && 
+                               export OPENAI_KEY=`aws secretsmanager get-secret-value --secret-id github_openai_key --query SecretString --output text` &&
+                               export COHERE_KEY=`aws secretsmanager get-secret-value --secret-id github_cohere_key --query SecretString --output text` &&
+                               echo "::add-mask::$OPENAI_KEY" &&
+                               echo "::add-mask::$COHERE_KEY" &&
+                               ./gradlew build &&
+                               ./gradlew publishToMavenLoca &&
+                               ./gradlew integTest -PnumNodes=3'
+          plugin=`basename $(ls plugin/build/distributions/*.zip)`
+          echo $plugin
+          mv plugin/build/distributions/$plugin ./
```

Comment:
Curious to know the output of `plugin/build/distributions/$plugin`

Reply:
I will change it to `mv -v` so you will see the entire output.